install/kubernetes/values.yaml

```diff
@@ -178,7 +178,23 @@ export:
   filenames:
   - tetragon.log
 
+  # stdout specific exporter settings
```

Comment:
While we're at it maybe we should just move extraArgs and extraEnv down here too?

Reply:
I moved extraEnv into export.stdout. I removed extraArgs since it is not used in the _container_export_stdout.tpl.